languages/csharp/Bitwarden.Sdk/Bitwarden.Sdk.csproj

```diff
@@ -0,0 +1,71 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+<PropertyGroup>
+  <TargetFramework>net6.0</TargetFramework>
+  <PackageReadmeFile>README.md</PackageReadmeFile>
+  <ImplicitUsings>enable</ImplicitUsings>
+  <Nullable>enable</Nullable>
+  <AssemblyName>BitwardenSdk</AssemblyName>
+  <PackageId>BitwardenSdk</PackageId>
```

Comment:
I don't think the packageId and AssemblyName are correct with the namespace changes?

Reply:
I will change it to Bitwarden.Sdk, but also there is packageId when package is created in git hub actions. I will change it there also.

Reply:
done